### ISSUE TITLE
Plotting ACF

### ISSUE THREAD

**Issue by mmd357** [EEDA-EES599/lab06-mmd357#2]
@nickmckay 

I'm not sure if I'm plotting this correctly.  Is there another way we are supposed to plot autocorrelation?

x1 = seq(1:1000)
d1 = arima.sim(list(order = c(1,0,0), ar = -.5), n = 1000)
d2 = arima.sim(list(order = c(1,0,0), ar = -.1), n = 1000)
d3 = arima.sim(list(order = c(1,0,0), ar = 0.01), n = 1000)
d4 = arima.sim(list(order = c(1,0,0), ar = .1), n = 1000)
d5 = arima.sim(list(order = c(1,0,0), ar = .3), n = 1000)
d6 = arima.sim(list(order = c(1,0,0), ar = .5), n = 1000)
d7 = arima.sim(list(order = c(1,0,0), ar = .7), n = 1000)
d8 = arima.sim(list(order = c(1,0,0), ar = .9), n = 1000)

AR1data = data.frame(x1, d1, d2, d3, d4, d5, d6, d7, d8)

ggplot(AR1data)+
  geom_point(aes(x1,d1))+
  geom_point(aes(x1,d2))+
  geom_point(aes(x1, d3))+
  geom_point(aes(x1, d4))+
  geom_point(aes(x1, d5))+
  geom_point(aes(x1, d6))+
  geom_point(aes(x1, d7))+
  geom_point(aes(x1, d8))+
  labs(x= "1:1000", y = "autocorrelation coefficient")

**Comment by nickmckay** [comment]
This looks reasonable to me..., although I would use lines of different
colors, so you can see the impact. That might be a bit much, so maybe split
it over two figures?

**Comment by mmd357** [comment]
@nickmckay 
Okay, I just didn't know if geom_point was the right way to go about it.  I didn't know if I needed to get "Lag" to be on the x-axis.  I wasn't seeing a major difference between negative and positive autocorrelation coefficients.  If I understand correctly, negative ACC would just mean they are having the same effect in the opposite direction, is that correct?

Thanks,
Megan

**Comment by nickmckay** [comment]
Yeah, use geom_line instead. And the x-axis should be time, or position in
the vector, not lag.

Also postitive and negative AR1 values should result in very different
looking data series, especially at high values.

Cheers,
Nick

**Comment by mmd357** [comment]
@nickmckay 
x1 = seq(1:1000)
d1 = arima.sim(list(order = c(1,0,0), ar = -.9), n = 1000)
d8 = arima.sim(list(order = c(1,0,0), ar = .9), n = 1000)

ggplot(AR1data)+
  geom_line(aes(x1,d1), color = "blue")+
  geom_line(aes(x1, d8), color= "red")+
  labs(x= "1:1000", y = "autocorrelation coefficient")

When I plot this, I'm having a hard time telling a big difference besides the fact that the negative AR1 coefficient seems to vary more around zero while the positive AR1 coefficient has a greater range.

I was also wondering if you had a paper you wanted us to read for #5.  I looked him up online and found a pdf of a paper from 1999, but didn't know if it was the right one.

Thanks

**Comment by nickmckay** [comment]
For those two you should see the negative ar value bouncing back and forth
around 0, whereas the positive takes long, consistent paths. Imagine
prediciting one value from the next. They both have predictability, but in
opposite ways.
